From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: deepak1556 <hop2deep@gmail.com>
Date: Fri, 5 Apr 2019 12:49:24 +0530
Subject: Define ipc identifier for legacy Electron ipc messages

Electron ipc messages used to piggy back on ShellMsgStart for ipc
identifier. But its not available after
https://chromium-review.googlesource.com/c/chromium/src/+/1525181.
This patch can be removed once legacy ipc messages are no longer
used.

diff --git a/ipc/ipc_message_start.h b/ipc/ipc_message_start.h
index ec19b740c8f27e2f8855d2ddbff5b33cd94d278e..79b1be79973aa875d860d797249f8c445ca1067a 100644
--- a/ipc/ipc_message_start.h
+++ b/ipc/ipc_message_start.h
@@ -70,6 +70,7 @@ enum IPCMessageStart {
   SubresourceFilterMsgStart,
   ChromeAppsMsgStart,
   UnfreezableFrameMsgStart,
+  ElectronMsgStart,
   LastIPCMsgStart  // Must come last.
 };
 
